src/purifier/README.md

```diff
@@ -0,0 +1,25 @@
+# AMP Purifier
+
+[![npm version](https://badge.fury.io/js/%40ampproject%2Fpurifier.svg)](https://badge.fury.io/js/%40ampproject%2Fpurifier)
```

Comment:
We use shields.io for badges in the repo's main readme file. For consistency, would it make sense to use it here too?

https://github.com/ampproject/amphtml/blame/master/README.md#L3-L34

Reply:
No problem, replaced with shields.io badge for npm package.